armsrc/Standalone/Makefile.hal

```diff
@@ -59,10 +59,13 @@ define KNOWN_STANDALONE_DEFINITIONS
 | HF_YOUNG        | Mifare sniff/simulation                |
 |                 | - Craig Young                          |
 +----------------------------------------------------------+
+| HF_AVEFUL       | Mifare ultralight read/simulation      |
```

Comment:
cf readme: Add your mode to the `Makefile.hal` help and modes list (alphabetically) ;)
same for the STANDALONE_MODES list.
thanks

Reply:
Sorry, I missed that one somehow. Should be good now.